.kitchen.yml

```diff
@@ -4,79 +4,72 @@ driver:
   provider: parallels
 
 provisioner:
-  product_name: chef
-  product_version: 13.8
+  name: chef_zero
```

Comment:
I think we should still be forcing the latest chef-client here, instead of what might be in a testing box.

Reply:
```
provisioner:
  product_name: chef
  product_version: latest
  install_strategy: always
```